_config/dev.yml

```diff
@@ -1,6 +1,9 @@
 ---
 Name: DevelopmentAdmin
 ---
+SilverStripe\Dev\BuildTask:
+  default_enabled: true
```

Comment:
Do we really need this `default_enabled` config?  Setting this to `false` would switch off all build tasks - that's outside the parameters of the original issue.  Seems like we should just omit this all together?

Reply:
Yeah I wasn't too sure either, if that is *needed*, but it just turned out quite convenient when I tried it in a project with a lot of tasks. So I figured I'd add just add it. 

It's easier to put your default to false and then setting two or three tasks to enabled, instead of having to disable a lot of tasks. 

I will just omit this for now, can still readd this or something similar in the future, if needed

Reply:
Just adding this here real quick, since you did not mark it; 

I believe we still need a fallback value if the config is not set *and* the task does not have the property $enabled set. 

I got some Tasks from addons that do not have it. Just came up when testing the changes

```php
    /**
     * @return bool
     */
    public function isEnabled()
    {
        $isEnabled = $this->config()->get('is_enabled');

        if ($isEnabled === null) {
            return static::$enabled ?? true;
        }
        return $isEnabled;
    }
```

I figured that `true` would be the smarter value as a fallback, lmk what you think

Reply:
We don't need this `static::$enabled ?? true;` - in CMS 5 `static::enabled` is still always there are. When we do CMS 6 we'll be paying attention to this method and will refactor it as needed to allows for `static_enabled` not being there

The core idea is that all BuildTasks use `static::$enabled` right now. Very few will support the new config value `$is_enabled`. `$is_enabled` has precedence over `static::$enabled`, so `static::$enabled` is the fallback.  No need for `?? true`